scanomatic/ui_server/data_api.py

```diff
@@ -16,9 +16,8 @@
 from scanomatic.data_processing.norm import NormState
 from scanomatic.data_processing.project import path_has_saved_project_state
 from scanomatic.image_analysis.first_pass_image import FixtureImage
-from scanomatic.image_analysis.grayscale import (
+from scanomatic.image_analysis.grayscale import \
```

Comment:
I guess we'll have to accept this, and the trailing `,` unless we can get the plugins to work the way we want. Unreasonable to fix these things by hand if the editor then comes to reset it! ;)

Reply:
I can't find it, but feel free to come up with a solution for the plugin

Reply:
What plugin are you using?